src/mlpack/core/util/timers.cpp

```diff
@@ -220,44 +134,16 @@ void Timers::StartTimer(const std::string& timerName)
 
   timerState[timerName] = true;
 
-  timeval tmp;
-  tmp.tv_sec = 0;
-  tmp.tv_usec = 0;
+  std::chrono::high_resolution_clock::time_point currTime = GetTime();
 
-  GetTime(&tmp);
-
-  // Check to see if the timer already exists.  If it does, we'll subtract the
-  // old value.
-  if (timers.count(timerName) == 1)
+  // If the timer is added first time
+  if(timers.count(timerName) == 0)
```

Comment:
Please add a space here between the if and its argument; I need to update the style guide to point this out.